articles/dd-you-did-what.md

```diff
@@ -0,0 +1,65 @@
+---
+authorName: Marko Ivanovic
+authorGithubUsername: ivma
+issue: 88
+title: You DiD what?
+---
+
+# {{page.title}}
+
+### Is it an outsourcing?! Is it a team sourcing?! No! It is the Distributed development!
+
+![DD image](./dd-you-did-what/dd-image.jpg)
+
+Dive into Distributed development topic and get all the information needed to kick off the project in Distributed Development Setup.
+
+### 1. What is it?
+Distributed development is a powerful way to realize project research, development, and realization across two or more physical locations. Difference between outsourcing and distributed development is that, in DD setup, all the organization is working together on the same level with a common goal of project realization. In Zuhlke Group, Distributed Development is a tool, a process, and a culture.
```

Comment:
Bare in mind that this article, once printed in a book, is for a broader audience than Zuhlke employees only. It will be publicly available. Therefore I suggest to remove the last sentence of this paragraph.

Reply:
Good point. I removed this line.